game-core/src/main/java/games/strategy/engine/framework/startup/ui/InGameLobbyWatcher.java

```diff
@@ -101,7 +101,7 @@ public static InGameLobbyWatcher newInGameLobbyWatcher(
     Preconditions.checkNotNull(handler);
     final @Nullable String host = getLobbySystemProperty(LOBBY_HOST);
     final @Nullable String port = getLobbySystemProperty(LOBBY_PORT);
-    final @Nullable String hostedBy = getLobbySystemProperty(LOBBY_GAME_HOSTED_BY);
+    final @Nullable String hostedBy = getLobbySystemProperty(TRIPLEA_NAME);
```

Comment:
See warning elsewhere about node names with ordinals.  This is assumed to be a node name.  If it doesn't match the player's node name, the associated lobby watcher may not work correctly.

Reply:
Hard to tell by inspection.  I'd try running a local lobby with one of the users logged in multiple times.  Have one of that user's nodes exercise some of the code in this class (e.g. watching [not playing] an in-progress game) and see if anything strange happens.

Reply:
testing looked good! thanks for the suggestion @ssoloff